src/js/visual/configurator.js

```diff
@@ -587,6 +587,7 @@ ripe.Configurator.prototype.lowlight = function(options) {
     const frontMask = this.element.querySelector(".front-mask");
     frontMask.classList.remove("highlight");
     this.element.classList.remove("highlight");
+    this.trigger("lowlight");

```

Comment:
If this event's counterpart is `highlighted_part` than this should be `lowlighted_part`.